notebooks/samples/102 - Regression Example with Flight Delay Dataset.ipynb

```diff
@@ -38,17 +38,17 @@
    "outputs": [],
    "source": [
     "# load raw data from small-sized 30 MB CSV file (trimmed to contain just what we use)\n",
-    "dataFile = \"On_Time_Performance_2012_9.csv\"\n",
+    "dataFilePath = \"On_Time_Performance_2012_9.csv\"\n",
     "import os, urllib\n",
-    "if not os.path.isfile(dataFile):\n",
-    "    urllib.request.urlretrieve(\"https://mmlspark.azureedge.net/datasets/\"+dataFile, dataFile)\n",
+    "if not os.path.isfile(dataFilePath):\n",
+    "    urllib.request.urlretrieve(\"https://mmlspark.azureedge.net/datasets/\" + dataFilePath, dataFilePath)\n",
     "flightDelay = spark.createDataFrame(\n",
-    "    pd.read_csv(dataFile, dtype={\"Month\": np.float64, \"Quarter\": np.float64,\n",
+    "    pd.read_csv(dataFilePath, dtype={\"Month\": np.float64, \"Quarter\": np.float64,\n",
     "                                 \"DayofMonth\": np.float64, \"DayOfWeek\": np.float64,\n",
```

Comment:
Reindent these lines too since the first one moved, or maybe drop the whole `dtype=` to the next line.